website_sale_product_brand/views/product_brand.xml

```diff
@@ -6,10 +6,10 @@
     <template id="product_brands" name="Brands">
         <t t-call="website.layout">
             <t t-set="additional_title">Product Brands</t>
-            <div id="wrap" t-attf-class="wrapper"> 
+            <div id="wrap" t-attf-class="wrapper">
```

Comment:
Yeah, good question. We need to see controller. No I'm changing my vote for not touching this, hehe. I would bet `t-att-class="wrapper"`, because although technically there's no `%` to format, it won't fail without them.